Makefile

```
@@ -30,7 +30,7 @@ fake:
 migrate:
 	docker-compose run app python manage.py migrate
 
-makemigrations:
+make migrations:
```

Hmm... `make make migrations`? Maybe it would be better to just name `migrations`.

```suggestion
migrations:
```

---

didnt realise that but `make migrations` is working for me 😕